CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/scatter/Scatter3dArranger.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.arrangements.scatter;
+
+import au.gov.asd.tac.constellation.graph.GraphElementType;
+import au.gov.asd.tac.constellation.graph.GraphWriteMethods;
+import au.gov.asd.tac.constellation.graph.schema.analytic.attribute.objects.RawData;
+import au.gov.asd.tac.constellation.graph.schema.visual.concept.VisualConcept;
+import au.gov.asd.tac.constellation.plugins.arrangements.Arranger;
+import au.gov.asd.tac.constellation.plugins.arrangements.GraphUtilities;
+import au.gov.asd.tac.constellation.utilities.color.ConstellationColor;
+import java.time.ZonedDateTime;
+import java.util.BitSet;
+
+/**
+ * Arrange the graph in a scatter3d.
+ *
+ * @author CrucisGamma
+ */
+public class Scatter3dArranger implements Arranger {
+
+    private final Scatter3dChoiceParameters params;
+
+    /**
+     * A scatter3d arrangement with default parameters.
+     */
+    public Scatter3dArranger() {
+        this(Scatter3dChoiceParameters.getDefaultParameters());
+    }
+
+    /**
+     * Construct new ArrangeInScatter3d instance.
+     *
+     * @param params Parameters for the arrangement.
+     */
+    public Scatter3dArranger(final Scatter3dChoiceParameters params) {
+        this.params = params;
+    }
+
+    @Override
+    public void setMaintainMean(final boolean b) {
```

Comment:
I see this is part of the `Arranger` interface - maybe just drop a comment in here about how it's not relevant to this arrangement.